src/Microsoft.AspNet.Diagnostics.Entity/DatabaseErrorPageOptions.cs

```diff
@@ -8,7 +8,7 @@ public class DatabaseErrorPageOptions
         public static DatabaseErrorPageOptions ShowAll => new DatabaseErrorPageOptions
                                                               {
                                                                   ShowExceptionDetails = true,
-                                                                  ListMigrations = true,
+                                                                  ListMigrations = true
```

Comment:
We prefer to keep the trailing comma wherever possible because it makes the diffs cleaner when items are added or removed from the list.